lib/imgur.js

```diff
@@ -86,7 +86,11 @@ imgur._imgurRequest = async (operation, payload, extraFormParams) => {
       break;
     case 'search':
       options.method = 'GET';
-      options.url += '/gallery/search/' + payload;
+      options.url += 'gallery/search/' + payload;
```

Comment:
This leading / doesn't break the request but it would break any tests we would want to create for this endpoint. I think this leading slash was left in by mistake, and after spending more time with this code I feel confident that is the case.

Reply:
i'm curious, in what way do tests break? is it bc of the way the url is matched in the handler?

Reply:
@kaimallea correct. It would be easy to fix, but then the URL would look weird:

`https://api.imgur.com/3//gallery/search/lDrXtHj`

Reply:
👍🏾